data_linter/validators/pandas_validator.py

```diff
@@ -406,17 +407,15 @@ def _check_meta_has_params(any_of: list, meta_col: dict):
     return any([a in meta_col for a in any_of])
 
 
-def _parse_data_to_pandas(filepath: str, table_params: dict, metadata: dict):
+def _parse_data_to_pandas(filepath: str, table_params: dict, metadata: Metadata):
     """
     Reads in the data from the given filepath and returns
     a dataframe
     """
 
     # get the required sets of column names
     meta_col_names = [
```

Comment:
could stick this line below the `if table_params.get("headers-ignore-case"):` bit, to avoid redefining it later

Reply:
It's needed on read in of the data in case there are no headers, unless I'm missing a cleaner way of doing it?

Reply:
so it is. can't think of anything cleaner so probably fine as-is